config.proto

```diff
@@ -266,6 +266,40 @@ message Config {
       ACCESS_POINT_HIDDEN = 2;
     }
 
+    enum ethMode {
```

Comment:
Can we make this enum name Pascal case as well?

Reply:
Yes, of course.